sum_odd_integers.rb

```diff
@@ -0,0 +1,25 @@
+
+# Enter at least two Integers
```

Comment:
Clearing up instruction.  

```
For this challenge, we will be writing a program that receives any amount of numbers from a user separated by spaces.  Once we have this string we will only want to return the sum of the odd numbers.  

For example, if the user were to enter "9 5 4" the program would only sum the "9" and the "5", because those are odd numbers, and return a sum of 14.  

The excepted output would be similar to this below: 

Enter at least 2 numbers separated by spaces: 
3 7 5 8 1 
16
```

Reply:
Thanks!

> that receives any amount of number

This should probably be "at least two Integers". I guess _technically_ it will work with one or zero inputs, but there are no tests for that. I also want to let them know that they should expect Integers and not Floats.

Reply:
I agree. I think your wording is better there.  Also, with some googling, did you know what all non-integrs are technically odd? I didn't know if a float could be even, and apparently, it cannot.

Reply:
> did you know what all non-integers are technically odd? 

That vaguely sounds familiar. I guess my concern is unwarranted. Maybe they could still enter a Float and if they use `to_i` they could get weird results? The tests wouldn't catch it though 🤷🏽